app/views/interview_questions/edit_multiple.html.slim

```diff
@@ -0,0 +1,47 @@
+- provide(:title, 'Interview')
+.container.content-area#workflow
+  h1.center
+	  | Interview: 
+	  = @interview.id
+  br
+  .form
+    = form_tag update_multiple_interview_questions_path, method: :put do 
+      #tabs
+        ul
+          - num=0
+          - @interview.questions.each do |q|
+            - num+=1
+            - l="li"+num.to_s
+            - hrf="#tabs-"+num.to_s
+            li.d-line id=l
+              a href=hrf
+                = num 
+        br
+        - num_tab=0
+        - @interview.questions.each do |q|
+          - hrf="tabs-"+(num_tab+1).to_s
+          div id=hrf
+            h3
+              | Question 
+              = num_tab+1
+              | :  
+              = q.content
+            h4
+              | Answer: 
+              = q.answer
+            br
+            - interview_question = InterviewQuestion.where(interview_id: @interview.id, question_id: q.id).first
```

Comment:
BTW any kind of assigning and defining of any kind of variables in views is not acceptable